physfs/psp.patch

```diff
@@ -0,0 +1,41 @@
+diff --git a/src/physfs.c b/src/physfs.c
+index fdb1405..7eaa5ba 100644
+--- a/src/physfs.c
++++ b/src/physfs.c
+@@ -11,6 +11,10 @@
+ #define __PHYSICSFS_INTERNAL__
+ #include "physfs_internal.h"
+ 
++#ifdef __PSP__
++#include <pthread.h>
++#endif
++
+ #if defined(_MSC_VER)
+ #include <stdarg.h>
+ 
+@@ -1194,6 +1198,9 @@ static int doDeinit(void);
+ 
+ int PHYSFS_init(const char *argv0)
+ {
++    #ifdef __PSP__
++    pthread_init();
```

Comment:
I really doubt this is still required. The need for initializing pthread was changed a while ago. Can you test without this change?

Reply:
I'll build without it tomorrow and check if it still works.

Reply:
I tested it, and it works! It works even with just a platform patch.

Reply:
Also tested now. `src/physfs.c` shouldn't be patched.

Reply:
Nice! I'll update my PR in a minute.